app/src/main/sentences/fr/current_time.dslf

```diff
@@ -0,0 +1,2 @@
+current_time: high
+quelle|(quelle est) <l'?>heure (est-il)?;
```

Comment:
- I don't know if this suggestion makes sense
- `quelle|(quelle est)` is equivalent to `quelle est?`
- apostrophes and dashes are not letters so they should not be used here. For simplicity in this languages words can only be made of letters. We don't care whether two words are separated by a dash/apostrophe; we only care for the fact that the two are separate words
```suggestion
quelle est? l? heure (il est)|(est il)?;
```

Reply:
Okay, the most used question will be "quelle heure est-il ? "

Reply:
Ok, I added `est il`, too, just because Deepl suggested me "quelle heure il est"